app/views/group_categories/edit.html.erb

```diff
@@ -0,0 +1,21 @@
+<h1 class="text-center">Edit Entity Category</h1>
+
+<%= form_for([@group_category, @entity_category], url: group_category_entity_category_path(@group_category, @entity_category), method: :patch) do |f| %>
+  <div class="field">
+    <%= f.label :name %>
+    <%= f.text_field :name %>
+  </div>
+
+  <!-- Add other form fields as needed -->
+   <div class="field">
+    <%= f.label :amount %>
+    <%= f.number_field :amount %>
+  </div>
+
+
```

Comment:
- [ ] As requested by the previous reviewer, please modify the form of creating new entities to include a form to choose at least one category to add to. 🙏

Reply:
@MahmoudBakr23 
Thanks so much for the feedback, I noticed you were in the wrong file and the correction has been made in the file path app/views/group_categories/index.html.erb and not in the edit, I will go ahead and clean the codes thanks again.

Reply:
@Zuwairanajma Thank you for communicating. I'm well aware of that. However, I selected a relevant form to just catch your eye on updating the live demo link with the requested changes you made and asked by the previous reviewer. Thanks! 👍